src/sql/workbench/contrib/backup/browser/backup.component.ts

```diff
@@ -447,13 +486,14 @@ export class BackupComponent extends AngularDisposable {
 			this.setDefaultBackupName();
 			this.backupNameBox!.focus();
 
-			// Set backup path list
-			this.setDefaultBackupPaths();
-			let pathlist: ISelectOptionItem[] = [];
-			for (let i in this.backupPathTypePairs) {
-				pathlist.push({ text: i });
+			if (this._engineEdition !== DatabaseEngineEdition.SqlManagedInstance) {
+				this.setDefaultBackupPaths();
```

Comment:
This logic looks quite similar to setBackupPathList - could you be using that instead with a bit of moving stuff around?

Reply:
I removed this logic since setBackupPathList already gets called in this method